packages/rum-core/test/common/apm-server.spec.js

```diff
@@ -368,16 +369,18 @@ describe('ApmServer', function() {
     configService.setConfig({
       serviceName: 'test',
       serviceVersion: '0.0.1',
-      environment: 'staging',
-      agentName: 'rum',
-      agentVersion: '3.0.0'
+      environment: 'staging'
     })
 
+    /** To catch agent version mismatch during release */
```

Comment:
I know we talked about this, but doesn't the tests run before the version change? meaning this test would pass anyway because the version change would happen after.

Reply:
Yeah you are right, We cannot rely on this test for releases. 

This would work for both our bundles as well as our consumers. WDYT?
```js
import . {version} from '../package.json' ?
```

Reply:
I think this won't work either since the build happens (as part of tests) before lerna changes the version.

I'm also not very fond of referencing package.json in our source code!